lib/artifact-search.js

```diff
@@ -124,14 +136,7 @@ module.exports = (bluebird, error, logger, requestOptions, requestPromise, respo
                     logger.debug(`Status code: ${response.statusCode}`);
                     urls = responseHandler.resolveLinkHeaders(response);
                     urls.forEach((val, index) => {
-                        var fullUrl;
-
-                        // Create a relative URL from the search and remove
-                        // the refName and "artifact" portion.
-                        fullUrl = new URI(responseHandler.resolveLink(options.url, val));
-                        fullUrl.segment(0, "");
-                        fullUrl.segment(0, "");
-                        urls[index] = fullUrl.path().toString();
+                        urls[index] = responseHandler.resolveLink(options.url, val);
```

Comment:
You've also added resolving the links inside of `responseHandler.resolveLinkHeaders()`, so we probably don't need to do it again.